manifests/params.pp

```diff
@@ -10,7 +10,11 @@
   $sources                = undef
   $puppet_master          = true
 
-  $r10k_basedir              = $facts['puppet_environmentpath']
+  if $facts['puppet_environmentpath'] {
```

Comment:
helidbe recommends to use different construct :

`if 'puppet_environment' in $facts`

Reply:
@hlindberg is there a technical reason for this? Is it a performance difference? The outcome at the end is the same.